packages/flutter/test/widgets/tween_animation_builder_test.dart

```diff
@@ -393,4 +393,195 @@ void main() {
     await tester.pump(const Duration(seconds: 2));
     expect(values, <Size>[const Size(10, 10)]);
   });
+
+  group('RepeatingTweenAnimationBuilder', () {
+    testWidgets('Repeats animation continuously', (WidgetTester tester) async {
+      final List<double> values = <double>[];
+      const Duration duration = Duration(milliseconds: 100);
+
+      await tester.pumpWidget(
+        RepeatingTweenAnimationBuilder<double>(
+          duration: duration,
+          tween: Tween<double>(begin: 0.0, end: 1.0),
+          builder: (BuildContext context, double value, Widget? child) {
+            values.add(value);
+            return const Placeholder();
+          },
+        ),
+      );
+
+      // Initial value should be 0.0
```

Comment:
Nit: could you do a pass on this file's comment and add periods at the end of all comments?